cinema_service/urls.py

```diff
@@ -2,10 +2,22 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularSwaggerView,
+    SpectacularRedocView
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("api/doc/swagger/",
+         SpectacularSwaggerView.as_view(url_name="schema"),
+         name="swagger-ui"),
+    path("api/doc/redoc/",
+         SpectacularRedocView.as_view(url_name="schema"),
+         name="redoc"),
 ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
The use of `static()` to serve media files in development is correct. However, ensure that a proper solution for serving static and media files is implemented for production environments.